### DISCUSSION TITLE
Installing saleor using plesk

### DISCUSSION THREAD

**Discussion by GoodGabriel** [saleor/saleor#7870] [Q&A]
Hi, i'm new to this stuff and i want to try (and use for my new company) saleor.
The problem is that I can't understand how to install saleor in my server.
I want to install different saleor sites in different subdomains and my server is using Plesk.
I saw and downloaded the docker plugin and downloaded saleor image but i cant understand how to link it to my subdomains

Hopefully someone can help me. Sorry for my bad english :c Thanks!

**Comment by karolkielecki** [comment]
Hi @GoodGabriel, 
I'm not sure if I understood you correctly. Do you want to run different projects (e-commerce stores) separately? Like "one Saleor" = one project? 

I'm not familiar with Plesk. As far as I remember, nobody has used Plesk to deploy Saleor before. Saleor's preferred deployment method is Docker. Storefront and dashboard are static apps so they don't require additional servers. You can use for them, for example, S3. https://docs.saleor.io/docs/3.0/developer/running-saleor/docker
